### ISSUE TITLE
Gene product display on GO-CAM page not ideal ? 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/web-gocam#14]
Hello, 

On the GO-CAM site https://geneontology.cloud/browse 
gene products show up as IRIs/links - is this intended ? 

I thought the gene name was shown 

<img width="697" alt="image" src="https://user-images.githubusercontent.com/4782928/102103977-f03e5780-3e2d-11eb-84f6-fb11ecce846c.png">

Thanks, Pascale

**Comment by lpalbou** [comment]
Yes, it was showing the gene labels before - at least when present in NEO.

I looked around and I think this is due to the changes of NEO made by Ben sometime ago. I fixed the query and we do get most gene names again.

There are however still a few genes that won't resolve:
![Screen Shot 2020-12-15 at 12 16 40 AM](https://user-images.githubusercontent.com/24249870/102188959-d0795480-3e6a-11eb-9880-b9f6a01fae60.png)

Interestingly, they do resolve in the graph editor: http://noctua.geneontology.org/editor/graph/gomodel:5e72450500001653 so this would indicate the triple store in production (rdf.geneontology.org) doesn't have the same NEO/gene info than minerva.

I think this is a sufficient fix as most gene names do resolve now, if that's the case, feel free to close that ticket.

**Comment by pgaudet** [comment]
Great, thanks for the quick response !

**Comment by lpalbou** [comment]
After discussion with bbop and to keep a reference of that xenbase gene that doesn't resolve with the blazegraph in production but does resolve within minerva (from noctua graph):
https://api.triplydb.com/s/oSw3hwZ2l

And as a counter example when it's work:
https://api.triplydb.com/s/zJub7DDmy

**Comment by balhoff** [comment]
@lpalbou how did you change the query to make it work? The problem is that NEO stopped being loaded into the graph store. I will fix that. But I'm curious what your change was, since as far as I know there should be no gene labels loaded in the current configuration.

**Comment by lpalbou** [comment]
@balhoff Ah Jim, sometimes I wonder what we actually have in our triple store ;) Like our friend Forrest Gump and its box of chocolate, you never know what you are gonna get with a schema less database.

[This was my fix](https://github.com/geneontology/api-gorest/commit/fbbe1dbcfadb0d447d118a0546cb30b47dfb10c8#diff-d7c232ee1a14d9c2f0cbf53068cff022d528fa52ac288cf52262a2cf535f2318). Instead of looking for the names in the Graph <http://purl.obolibrary.org/obo/go/extensions/go-graphstore.owl>, I started looking at the rool level. I didn't do some archeology to find out from which graph the data came from.

Thanks for https://github.com/geneontology/go-site/issues/1599.
